nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/resources/docs/org.apache.nifi.processors.mongodb.PutMongoRecord/additionalDetails.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>PutMongoRecord</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+</head>
+
+<body>
+<!-- Processor Documentation ================================================== -->
+<h2>Description:</h2>
+<p>
+    This processor is a record-aware processor for inserting data into MongoDB. It uses a configured record reader and
+    schema to read an incoming record set from the body of a flowfile and then inserts batches of those records into
+    a configured MongoDB collection.
+</p>
+<p>
+    This processor does not support updates, deletes or upserts. The number of documents to insert at a time is controlled
+    by the <em>Insert Batch Size</em> configuration property. This value should be set to a reasonable size to ensure
```

Comment:
Nitpick: Since these are all text and we are not actually introducing any cool HTML formatting, would it not be enough if we just add it to the processor's `@CapabilityDescription` itself?

Reply:
I agree, although Additional Details pages are always helpful, in this case I think this information is more important to be on the "front page", and any Additional Details (such as examples) would go here.